functions/src/glean.ts

```diff
@@ -5,7 +5,7 @@ import functions from "firebase-functions";
 import { Mutex, Semaphore, withTimeout } from "async-mutex";
 import fs from "fs";
 
-const RWP_pkg = JSON.parse(fs.readFileSync("../package.json").toString());
+const RWP_pkg = JSON.parse(fs.readFileSync("package.json").toString());
```

Comment:
@aaga I'm not sure why this started failing in my branch and not in https://github.com/mozilla-rally/rally-web-platform/commit/91f49efcb7f2adc7212e87673ebb4070a4ea0c19 when you landed this

Reply:
Oh! I think it is failing, and before this because of a package-lock sync problem (which I think I happened to fix in this branch):

https://app.circleci.com/pipelines/github/mozilla-rally/rally-web-platform/748/workflows/de47ce15-3a66-4bbe-bf8a-ca8c8f6c7a87/jobs/1059
```
pm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR! 
npm ERR! 
npm ERR! Invalid: lock file's firebase-tools@9.23.3 does not satisfy firebase-tools@^10.7.1
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2022-04-27T18_09_22_308Z-debug.log
```